### PR TITLE
Added OCP 3.5 Metrics Recommendations to the Enabling Cluster Metrics doc

### PR DESCRIPTION
**Pull request by ahardin-rh**
This continues the work from https://github.com/openshift/openshift-docs/pull/4019.

Related information was placed in the OCP 3.5 Release Notes via https://github.com/openshift/openshift-docs/pull/4046

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@ekuric PTAL

cc @mffiedler @jeremyeder 

@adellape @bfallonf Please peer review

**Comment by bfallonf** [comment]
@ahardin-rh I think this is all good :star: 

Should I attempt to include this in the scaling book we're working on do you think? (After this is merged of course)

**Comment by ahardin-rh** [comment]
@bfallonf Yes, thank you. After looking at your PR https://github.com/openshift/openshift-docs/pull/3881, I think we can move these bits into you new guide.

**Comment by bfallonf** [comment]
@ahardin-rh Neato. I'll let you merge away if this is ready, and I can incorporate it then.

**Comment by ahardin-rh** [comment]
@ekuric @jeremyeder @mffiedler PTAL
This will be moved into Brice's  new guide in https://github.com/openshift/openshift-docs/pull/3881 after we merge

**Comment by ekuric** [comment]
@ahardin-rh looks ok, if we can add there also sentence about 25000 pods we got in v3.5

this :  https://github.com/ekuric/openshift-docs/commit/c56b42891a18f6a1c9483b1d8cfec4135c88c331#diff-f45df46140efd0b1f090f2d3bd06ba04R14

**Comment by ekuric** [comment]
@ahardin-rh if we can include here sentence that we were able to monitor 25000 pods in v3.5 

https://github.com/ekuric/openshift-docs/commit/c56b42891a18f6a1c9483b1d8cfec4135c88c331#diff-f45df46140efd0b1f090f2d3bd06ba04R14

File: install_config/cluster_metrics.adoc [hunk 1]
```diff
@@ -263,7 +264,24 @@ link:http://docs.datastax.com/en/archived/cassandra/1.2/cassandra/architecture/a
 documentation].
 ====
 
-*Known Issues and Limitations*
+[discrete]
+[[metrics-recommendations-for-OCP-version-35]]
+==== Recommendations for {product-title} Version 3.5
+
+* Run metrics pods on dedicated {product-title} nodes.
```
Comment (jeremyeder) [inline install_config/cluster_metrics.adoc]: dedicated nodes should be clarified to be the infra nodes.

File: install_config/cluster_metrics.adoc [hunk 2]
```diff
@@ -263,7 +264,24 @@ link:http://docs.datastax.com/en/archived/cassandra/1.2/cassandra/architecture/a
...
+* Closely monitor {product-title} nodes with host metrics pods to detect early
+capacity shortages (CPU and memory) on the host system. These capacity shortages
+can cause problems for metrics pods.
+
+[discrete]
+[[cluster-metrics-known-issues-and-limitations]]
+==== Known Issues and Limitations
 
 Testing found that the `heapster` metrics component is capable of handling up to
 12000 pods. If the amount of pods exceed that number, Heapster begins to fall
```
Comment (jeremyeder) [inline install_config/cluster_metrics.adoc]: this 12000 number needs to be updated to 25,000.

**Comment by ahardin-rh** [comment]
@jeremyeder @ekuric Thanks! This is now updated. I will merge it and @bfallonf can move the information into his guide. :rocket: 

FYI @vikram-redhat since you will be cherry-picking this for the OCP 3.5 release.

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../install_config/cluster_metrics.adoc#install-config-cluster-metrics[Enabling Cluster Metrics]
|Added metrics recommendations for {product-title} version 3.5.
%

**Pull request merged**